pythonscript/embedded/godot/hazmat/lazy_bindings.py

```diff
@@ -230,10 +230,34 @@ def bind(self, *args):
         rettype = meth['return']['type']
 
         def bind(self, *args):
-            if len(args) != len(meth['args']):
-                raise TypeError('%s() takes %s positional argument but %s were given' %
-                                (methname, len(meth['args']), len(args)))
-            # TODO: check args number and type here (ptrcall means segfault on bad args...)
+            # check number of args
+            n_args, nm_args, nmd_args = len(args), len(meth['args']), len(meth['default_args'])
+            nr_args = nm_args - nmd_args    # number of required arguments
+            if n_args < nr_args:  # not enough args, raise error
+                if nr_args - n_args == 1:
+                    raise TypeError("%s() missing 1 required positional argument: '%s'" %(
+                                    methname, meth['args'][nr_args-1]['name'])
+                                    )
```

Comment:
over-indented closing parenthesis

Reply:
From [PEP-8](https://www.python.org/dev/peps/pep-0008/#indentation):
>The closing brace/bracket/parenthesis on multiline constructs may either **line up under the first non-whitespace character of the last line** of list, [...] or it may be lined up under the first character of the line that starts the multiline construct.

but I actually don't mind, I'll set it as you like but some [guidelines for contribution](https://blog.github.com/2012-09-17-contributing-guidelines/) would help :sweat_smile: (when you've got some time of course)

Reply:
PS: I bring up PEP-8 because you pointed to it in the other change request

Reply:
PS2: I guess you want me to follow the second alternative:
> [...] or it may be lined up under the first character of the line that starts the multiline construct.

Reply:
I would have written it this way:
```python
raise TypeError("%s() missing 1 required positional argument: '%s'" % (
    methname, meth['args'][nr_args-1]['name'])
)
```
However it's very possible my own way of writting is not as PEP8 as I would think 😄 
I should setup yapf to settle this kind of question, so in the meantime I won't bother as long as the dev coded with PEP8 in mind ^^